build.sbt

```diff
@@ -14,7 +14,8 @@ incOptions := incOptions.value.withNameHashing(true)
 
 libraryDependencies ++= Seq(
     "com.kinja" %% "amqp-client" % "1.5.0",
-    "com.typesafe.play" %% "play-json" % "2.3.4"
+    "com.typesafe.play" %% "play-json" % "2.3.4",
+    "com.typesafe.slick" %% "slick" % "1.0.1"
```

Comment:
what if we don't use the lifted slick syntax, but direct sql? then we just specify the version number as 1+, and hope that the direct sql syntax don't change from version to version :)